src/main/resources/com/google/api/codegen/metadatagen/java/grpc/package_staging.snip

```diff
@@ -0,0 +1,59 @@
+@snippet generate(metadata)
```

Comment:
What makes it so that we can't use a single build.gradle for both staging & releasing?